requirements.txt

```diff
@@ -2,18 +2,24 @@
 # This file is autogenerated by pip-compile
 # To update, run:
 #
-#    pip-compile --output-file requirements.txt requirements.in
+#    pip-compile
 #
+aiocontextvars==0.2.2     # via loguru
 aniso8601==1.2.1          # via flask-restful, flask-restplus
+appdirs==1.4.3            # via pyppeteer
 apscheduler==3.5.0
 babelfish==0.5.5          # via guessit
 beautifulsoup4==4.6.0
+bs4==0.0.1                # via requests-html
```

Comment:
Hmm. This is a dummy package, I don't see it directly in requests-html setup.py file, I wonder where it's getting added.

Reply:
maybe a pip-tools bug? kinda doubt it